src/rgw/rgw_lc.cc

```diff
@@ -1990,6 +1990,12 @@ int RGWLC::process(LCWorker* worker,
     }
   }
 
+  ret = static_cast<rgw::sal::RadosStore*>(driver)->getRados()->process_expire_objects(this, null_yield);
+  if (ret < 0) {
+    ldpp_dout(this, 5) << "RGWLC::process_expire_objects: failed, "
+	          << " worker ix: " << worker->ix << dendl;
+  }
```

Comment:
we need to return ret from here, or just log without error is enough?

Reply:
IMO we cannot fail main LC process for errors in expiring these temporary objects..